packages/api-browser/src/window.ts

```diff
@@ -214,5 +222,6 @@ const eventMap = {
   'focus': 'focus',
   'hide': 'hidden',
   'message': 'message',
-  'show': 'load'
+  'show': 'load',
+  'capture': 'capture'

```

Comment:
I think we probably don't need this - these are mappings for window event names.